alliance/core/lib/views_helper.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import _get_queryset
+from django.core.exceptions import MultipleObjectsReturned
+from django.db.models.Model import DoesNotExist
```

Comment:
This import fails. We can remove this line and use the DoesNotExist exception from the model we are trying to get (see below)